src/panel/direction/direction_panel.js

```diff
@@ -22,9 +20,18 @@ export default class DirectionPanel {
     this.origin = null
     this.destination = null
     this.vehicle = this.vehicles.DRIVING
-    this.roadMapPanel = new RoadMapPanel()
+    this.roadMapPanel = new RoadMapPanel(this)
```

Comment:
Would it be possible to register a callback instead of passing `this` ? I find a bit dangerous to let roadMapPanel handle the instance

Reply:
I mean, a callback to handle a close event

Reply:
good question, i find it more elegant than register callback and let the panel handle his own close method.